graphql/version_resolver.go

```diff
@@ -141,7 +141,7 @@ func (r *versionResolver) ChildVersions(ctx context.Context, obj *restModel.APIV
 func (r *versionResolver) ExternalLinksForMetadata(ctx context.Context, obj *restModel.APIVersion) ([]*ExternalLinkForMetadata, error) {
 	pRef, err := data.FindProjectById(*obj.Project, false, false)
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding project `%s`: %s", *obj.Project, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding project `%s`: %s", *obj.Project, err.Error()))
```

Comment:
use normal quotes, and `utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding project '%s': %s", *obj.Project, err.Error()))
```